src/components/pages/auth/login/index.tsx

```diff
@@ -13,6 +13,14 @@ import { toastify } from "@/helper/toastify";
 import { loginUser } from "@/backend/auth.api";
 
 export default function LoginComponent() {
+  function showPassword(): void {
```

Comment:
Is it possible to use html `event` instead of using `getElementById` for updating.